src/applications/vaos/components/InfoAlert.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function InfoAlert({
+  backgroundOnly = false,
+  children,
+  headline,
+  level = 2,
+  status,
+}) {
+  const H = `h${level}`;
+  if (backgroundOnly) {
+    return (
+      <div
+        className={`usa-alert usa-alert-${status} background-color-only vads-u-display--block`}
+      >
+        <div className="usa-alert-body">
+          {headline && (
+            <H className="usa-alert-heading vads-u-font-size--h3">{headline}</H>
+          )}
+          <div className="usa-alert-text">{children}</div>
+        </div>
+      </div>
+    );
+  }
+  return (
+    <va-alert class="vads-u-margin-top--3" status={status}>
+      <H className="vads-u-font-size--h3" slot="headline">
+        {headline}
+      </H>
+      <div className="vads-u-font-size--base">{children}</div>
```

Comment:
Is this font size adjustment going to be required for all of our alerts? It could be worth bringing up to the DS team if so, since it seems odd that we have to do that.

Reply:
Good idea. I'll create a ticket for the DS team to review